packages/german-words/0.0.3/README.md

```diff
@@ -0,0 +1 @@
+A Package that inclues German words :WORD for lowercase and !WORD for uppercase. View Online [Documentation](https://espanso.org/docs/get-started/)
```

Comment:
Before you go too far with separate lowercase and uppercase entries, are you sure they're preferable to [`propagate-case:`](https://espanso.org/docs/matches/basics/#case-propagation) keys?

I presume you'll be updating this file at some point, so I won't interfere more than necessary at this point.